spec/order_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'order'
+
+describe Order do
+
+  subject(:order) { described_class.new(menu_klass = menu, order_summary_klass\
+  = OrderSummary.new, sms_klass = sms)}
+  let(:sms) { double('sms')}
+  let(:menu) { double('menu', :dishes => { "Golden French Fries" => 0.5 }) }
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)